icu4j/maven-migration/toMaven.sh

```diff
@@ -226,14 +260,27 @@ function moveCircDepTestOutOfCore() {
 
   # Looks like the packaging project was already some kind of test for how things come together.
   # Should we move all the files in this project instead of common_tests?
-  mv main/tests/packaging/src/com/ibm/icu/dev/test/* $COMMON_TEST_FOLDER/
+  moveOrRename main/tests/packaging/src/com/ibm/icu/dev/test/* $COMMON_TEST_FOLDER/
   removeEclipseProjectFiles main/tests/packaging
 
   # At this point this folder should be empty
   # So remove if empty (-d) should work
   rm -d main/core/src/test/java/com/ibm/icu/dev/test/serializable
 }
 
+function moveToolsFile() {
```

Comment:
Nitpick: combine the 2 functions into one?
(`moveFilesInTools` and `moveToolsFile`)?
And comment.
I understand it is a matter of taste...
But I think it would be more readable if it's all in one, with comments.

Reply:
Done.